djstripe/models/billing.py

```diff
@@ -1825,28 +1847,71 @@ class TaxId(StripeModel):
     verification = JSONField(help_text="Tax ID verification information.")
 
     def __str__(self):
-        return self.value
+        return f"{enums.TaxIdType.humanize(self.type)} {self.value} ({self.verification.get('status')})"
```

Comment:
self.type.humanize() doesn't work? But also when was that method introduced?

Reply:
I just checked. It does work. It was added [here](https://github.com/dj-stripe/dj-stripe/commit/9e8d090b8929bf1fa40921dd2d99f7ff3bb794dd).